build.gradle

```diff
@@ -164,202 +162,192 @@ subprojects {
         }
     }
 
-    tasks.withType(Test) {
-        // Getting javadoc to compile is part of the test suite to ensure we are able to publish our artifacts
-        dependsOn project.javadoc
-    }
-    if (!sourceSets.test.allSource.files.isEmpty()) {
-        tasks.withType(Test) {
-            testLogging {
-                events "passed", "skipped", "failed"
-                exceptionFormat "full"
-                showExceptions true
-                showCauses true
-                showStackTraces true
-            }
-            maxParallelForks = gradle.startParameter.maxWorkerCount
+    plugins.withId('java') {
 
-            // Provide way to exclude particular tests from CLI
-            // e.g. ../gradlew test -PexcludeTests=**/KafkaProtobufConsumerLongTermTest*
-            if (project.hasProperty('excludeTests')) {
-                exclude project.property('excludeTests')
+        java {
+            toolchain {
+                languageVersion.set(JavaLanguageVersion.of(17))
+                vendor.set(JvmVendorSpec.AMAZON)
             }
+        }
 
-            useJUnitPlatform()
-            //  Disable parallel test execution, see MIGRATIONS-1666
-            systemProperty 'junit.jupiter.execution.parallel.enabled', 'false'
-            systemProperty 'log4j2.contextSelector', 'org.apache.logging.log4j.core.selector.BasicContextSelector'
-
-            // Register BucketTestExtension for all tests to enable test striping
```

Comment:
The `test.striping` changes are needed for GHA to be able to run sub-sets of our tests cases - that's why the GHA runtimes are so large, each runner is executing all the tests.

Reply:
Thanks.  I missed those in the merge & have brought them back.